packages/runtime/src/next/client.ts

```diff
@@ -134,13 +135,15 @@ export class Makeswift {
   private apiOrigin: URL
   private graphqlClient: GraphQLClient
   private runtime: ReactRuntime
-  private siteVersion: MakeswiftSiteVersion | null
+  private versioning: boolean
+  private siteVersion: MakeswiftSiteVersion
 
   constructor(
     apiKey: string,
     {
       apiOrigin = 'https://api.makeswift.com',
       runtime = ReactRuntime,
+      versioning = false,
```

Comment:
I remember that the reason why I thought it would be weird to pass both `versioning` and `unstable_previewData` here is that it allows for an invalid state. If `versioning` is `false` but `unstable_previewData` has a version, like `'Working'` or `'Live'`, then its inconsistent. `unstable_previewData` should by itself be enough to know, no? If it's not provided, i.e., `undefined` then `isUsingVersioning` should be `false`. If it's `null` then it's a live page so `isUsingVersioning` is `true` and `siteVersion` is `'Live'`, otherwise it's whatever version is defined by `unstable_previewData`.

I know this is contrary to what we had talked about, but I'm realizing it again after seeing `versioning` side by side with `unstable_previewData`.

Is the plan to remove the `unstable_` APIs? If so, are we not removing `unstable_previewData`? If we do remove `unstable_previewData`, I think we can come up with an API that is a single field that is enough for us to determine both `isUsingVersioning` and `siteVersion`